### PR TITLE
 CBG-164: Follow-up work for Iridium DCP stats

### PR DESCRIPTION
**Pull request by JRascagneres**
- [x] Merge https://github.com/couchbaselabs/walrus/pull/48
- [x] Merge https://github.com/couchbase/sg-bucket/pull/47
- [x] Bump manifest
http://uberjenkins.sc.couchbase.com:8080/job/sync-gateway-integration-master/1178/display/redirect

### CODE REVIEW

File: base/dcp_feed.go [hunk 1]
```diff
@@ -279,10 +283,7 @@ func (r *DCPReceiver) GetMetaData(vbucketId uint16) (
 // it's called, logs warning and does a hard reset on metadata for the vbucket
 func (r *DCPReceiver) Rollback(vbucketId uint16, rollbackSeq uint64) error {
 	Warnf(KeyAll, "DCP Rollback request.  Expected RollbackEx call - resetting vbucket %d to 0.", vbucketId)
-
-	// TODO: move to per-db stats, or if that's too difficult due to package dependencies, then at least under Global stats
-	dcpExpvars.Add("rollback_count", 1)
-
+	r.dbStatsExpvars.Add("dcp_rollback_count", 1)
```
Comment (adamcfraser) [inline base/dcp_feed.go:280]: It looks like there are still some usages of the previous dcpExpvars remaining - any reason those shouldn't be migrated along with these to the new location?

**Review by adamcfraser — CHANGES_REQUESTED** [review]
Having to pass an expvar.map all the way down the stack to the DCP feed is a bit awkward, but I don't see an obvious, cleaner way to do it.  Ideally we'd want to remove all of the old-style expvars - added a question about that.

File: base/dcp_feed.go (hunk 1 above)
Comment (JRascagneres) [reply base/dcp_feed.go:280]: Not really. Just that was the only one commented. I guess the others fit fine in the per-db too?

Comment (bbrks) [reply base/dcp_feed.go:280]: @JRascagneres yeah all can go under the per-db stats

File: base/dcp_feed.go [hunk 2]
```diff
@@ -760,7 +754,7 @@ func (b *backfillStatus) updateStats(vbno uint16, previousVbSequence uint64, cur
 	b.receivedSequences += backfillDelta
 
 	// NOTE: this is a legacy stat, but cannot be removed b/c there are unit tests that depend on these stats
-	dcpExpvars.Add("backfill_completed", int64(backfillDelta))
+	b.dcpReceiver.dbStatsExpvars.Add("dcp_backfill_completed", int64(backfillDelta))
```
Comment (adamcfraser) [inline base/dcp_feed.go:757]: Looks like we'll need to update TestDcpBackfill with this change.

**Review by adamcfraser — CHANGES_REQUESTED** [review]

**Pull request merged**